programs/mmm/src/instructions/vanilla/sol_fulfill_sell.rs

```diff
@@ -143,25 +143,22 @@ pub fn handler<'info>(
         args.allowlist_aux,
     )?;
 
-    let (total_price, next_price) =
-        get_sol_total_price_and_next_price(pool, args.asset_amount, false)?;
-    let lp_fee = get_sol_lp_fee(pool, buyside_sol_escrow_account.lamports(), total_price)?;
-
-    assert_valid_fees_bp(args.maker_fee_bp, args.taker_fee_bp)?;
-    let maker_fee = get_sol_fee(total_price, args.maker_fee_bp)?;
-    let taker_fee = get_sol_fee(total_price, args.taker_fee_bp)?;
-    let referral_fee = u64::try_from(
-        maker_fee
-            .checked_add(taker_fee)
-            .ok_or(MMMErrorCode::NumericOverflow)?,
-    )
-    .map_err(|_| MMMErrorCode::NumericOverflow)?;
-
-    let transfer_sol_to = if pool.reinvest_fulfill_sell {
-        buyside_sol_escrow_account.to_account_info()
-    } else {
-        owner.to_account_info()
-    };
+    let PoolPriceInfo {
```

Comment:
just want to double check if we cover all the necessary logic within this function refactoring

Reply:
yeah it's simply a DRY refactor and should be covered by the test